### ISSUE TITLE
About mAP &AP 50

### ISSUE THREAD

**Issue by weiguangzhao** [hustvl/HAIS#6]
Thank you for your contribution, this is a great job. First of all, there is no offense, whether the test set is simpler than the verification set? because test set AP50 is 69.9, and the verification set AP50 is 64.4. I'm currently studying the semantics and instance segmentation of ScannetV2, but I can only upload the results every two weeks. So I want to know if I missed something which can improve the accuracy on the test set.  Thank you.

**Comment by weiguangzhao** [comment]
Thank you

**Comment by weiguangzhao** [comment]
Hi, thank you for the previous answer. I find some methods(Pointgroup DyCo3D) have this gap, but some methods(3D-MPA) don't.  I use my method to test it and didn't get the gap. When you get the highest value on validation set, have you used the validation set to train again? I find some methods will do this, if that, how to get the suitable pre-trained model to test the test set? Thank you so much

**Comment by outsidercsy** [comment]
For the test set results, we do include the val set for training, which is a common practice. The test set submission is restricted. We just tune hyper-parameters on the trainval sets, and then adopt the same hyper-parameters to report the test test results only once. We think the pre-trained model is not important. Training from scratch is also OK if hyper-parameters are well tuned.

**Comment by weiguangzhao** [comment]
Thank you so much.
